src/AddIns/Uno.UI.Lottie/LottieVisualSource.cs

```diff
@@ -6,7 +6,7 @@
 using Windows.UI.Xaml.Data;
 using Uno.Disposables;
 
-namespace Microsoft.Toolkit.Uwp.UI.Lottie
```

Comment:
That needs to be dependent on UWP vs. WinUI, I would think.

Reply:
I'll adjust.